browser/ipfs/ipfs_tab_helper.cc

```diff
@@ -62,19 +111,115 @@ bool IPFSTabHelper::MaybeCreateForWebContents(
   return true;
 }
 
+void IPFSTabHelper::HostResolvedCallback(
+    const std::string& host,
+    const std::vector<std::string>& text_results) {
+  GURL current = web_contents()->GetURL();
+  if (current.host() != host || !current.SchemeIsHTTPOrHTTPS())
+    return;
+  std::string dnslink = GetDNSRecordValue(text_results, kDnsLinkHeader);
+  if (dnslink.empty())
```

Comment:
we don't do anything with this value? I think this needs a comment explaining what we're doing here

Reply:
yes, we should only check if it exists, https://github.com/brave/brave-browser/issues/13609#issuecomment-790030606
I will add a comment